installer/scripts/acs-integration-helper.sh

```diff
@@ -92,11 +92,13 @@ get_variables() {
     fi
 
     if [ -z "${REGISTRY_ENDPOINT:-}" ]; then
-        REGISTRY_ENDPOINT="$(oc get secrets -n "$NAMESPACE" "$REGISTRY_SECRET" -o json | jq -r '.data.url | @base64d')"
-    fi
-    if [ -z "${REGISTRY_HOST:-}" ]; then
-        REGISTRY_HOST="${REGISTRY_ENDPOINT//https:\/\//}"
+        REGISTRY_ENDPOINT="$(
```

Comment:
That should not be needed. Is the `url` missing when using some integrations? If that is the case then the integrations should be fixed.

Reply:
No, the `url` is not missing, the `url` is console url when using integration with Nexus or Artifactory. I don't think It's a bug. I verified that  if the url is the registry endpoint when integrating with Nexus or Artifactory, images cannot show up in Image Registry page.

Reply:
I am confused why `REGISTRY_ENDPOINT ` should not be needed. It needs to be passed into the data body as the value of endpoint.
```
{
    "autogenerated": false,
    "categories": ["REGISTRY"],
    "clusterId": "",
    "id": "",
    "name": "'"${REGISTRY_ENDPOINT}"'",
    "'"${INTEGRATION_TYPE}"'": {
        "endpoint": "'"${REGISTRY_ENDPOINT}"'",
        "insecure": '"$INSECURE"',
        '"$REGISTRY_CREDENTIALS"'
    },
    "skipTestIntegration":false,
    "type":"'"${REGISTRY_TYPE}"'"
}
```

Reply:
REGISTRY_ENDPOINT is needed.

My understanding is that REGISTRY_ENDPOINT should match the `url` of the integration secret, but you are saying `I verified that if the url is the registry endpoint when integrating with Nexus or Artifactory, images cannot show up in Image Registry page.`, so I'm obviously wrong.

So now, I'm trying to understand how the `url` and `.auth` values differ. Can you show me an example for each of the image registry?

Reply:
Added comments to explain it